test/ng/jsonpCallbacksSpec.js

```diff
@@ -2,11 +2,6 @@
 
 describe('$jsonpCallbacks', function() {
 
-  beforeEach(module(function($provide) {
-    // mock out the $window object
-    $provide.value('$window', { angular: {} });
-  }));
```

Comment:
There is already code in [angular-mocks][1] that cleans up `angular.callbacks`. This isn't necessary.

[1]: https://github.com/angular/angular.js/blob/f6c3b357451a2125550811946084fc25ae4eb0ce/src/ngMock/angular-mocks.js#L2954-L2957